config/checkstyle/suppressions.xml

```diff
@@ -10,10 +10,9 @@
     <suppress checks="Javadoc.*" files=".*Test.java"/>
     <suppress checks="MethodName" files=".*Logger.java"/>
     <suppress checks="MethodName" files=".*LoggerTest.java"/>
-    <suppress checks="MemberName" files="AuthenticationContext.java" lines="813"/>
-    <suppress checks="VisibilityModifier" files="AuthenticationContext.java" lines="813"/>
     <suppress checks="MemberName" files="SSOStateSerializer.java" lines="47"/>
     <suppress checks="HideUtilityClassConstructor" files="ReflectionUtils.java" lines="31"/>
```

Comment:
this is not related to your change, but can you remove all the line number here? this is not going to scale, if we do code changes, it's so easy to break.